tests/test_shelf.py

```diff
@@ -542,6 +542,18 @@ def test_null_condition(self):
         with pytest.raises(Exception):
             self.make_shelf(content)
 
+    def test_null_aggregation(self):
+        content = '''
+oldage:
+    kind: Metric
+    field:
+        value: age
+        aggregation: null
```

Comment:
This is a change in behavior but I think it's reasonable. If someone explicitly requests a metric with no aggregation, we now support it. It's useful for some cases where you just want to get and show rows of data without doing any aggregation. The TDOE is like this.